guides/V1_ActionsEvents.md

```diff
@@ -0,0 +1,199 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'actions-and-events'
+is: 'experimental'
+---
+## Actions and Events
+
+Watson Work Services gives your App two patterns for dealing with events, which take place either due to changes in the data model and/or due to actions humans take using a participating client in Watson Work Services such as the Workspace clients. 
+
+Your App can Listen to Events, through secure web hooks. Also, your App can handle actions that may be invoked 
+from a Watson Work Services client so that you can provide an experience to users for taking that action, in two ways, 
+using Client Action Handlers or user entered Commands. 
+
+Note: This will introduce a change to how to make your app configurable. Currently, in the [Your Apps section](https://developer.watsonwork.ibm.com/apps) there is a section called **Additional Information** where you enter a configuration URL (for configuring your app in a space) and Terms of Services for your app. This change we will move Terms of Service to it's own section, and remove the **Additional Information** section. The rest of this guide explains what you will do to set up configuration. If you had already establised an app with configuration, you will not need to do anything to continue working as is, and can update your app as needed. 
+
+## Listen to Events
+
+Please read [Listen to Events](https://github.com/watsonwork/watsonwork-developer-docs/blob/master/guides/V1_wwsg_Webhooks.md) here.
+
+## Handling Client Actions
+
+Watson Work Services allows your App to provide for actions issued from a Watson Work Services client (like Watson Workspace). 
```

Comment:
>Watson Work Services allows your App to provide for actions issued from a Watson Work Services client (like Watson Workspace).

to:

Watson Work Services allows your App to provide a user experience for actions issued from a Watson Work Services client (like Watson Workspace)

Reply:
> Example: App Configuration

Example: App Configuration, Sharing Connections Files resources